packages/tc-ui/src/pages/edit/browser.js

```diff
@@ -1,9 +1,17 @@
 require('./main.css');
+const { init } = require('@financial-times/tc-schema-sdk');
 const {
 	Relationship: { withEditComponent: attachRelationshipPicker },
 	LargeText: { withEditComponent: attachDocumentEditor },
 } = require('../../primitives/browser');
 
+const initSchema = async () => {
+	const jsonNode = document.querySelector("[data-json='schema-data']");
+	const jsonRaw = jsonNode && jsonNode.textContent;
+	const jsonData = jsonRaw && jsonRaw.length > 4 ? JSON.parse(jsonRaw) : {};
```

Comment:
Why checking for `jsonRaw.length > 4` ?

Reply:
no particular reason other than guarding 'null' string